IdentityCore/tests/integration/ios/MSIDDefaultSilentTokenRequestTests.m

```diff
@@ -340,6 +341,115 @@ - (void)testAcquireTokenSilent_whenExpiredAccessTokenInCache_shouldRefreshToken
     }];
 
     [self waitForExpectationsWithTimeout:1.0 handler:nil];
+
+    MSIDAccessToken *accessToken = [tokenCache getAccessTokenForAccount:accountIdentifier configuration:silentParameters.msidConfiguration context:nil error:nil];
+    XCTAssertNotNil(accessToken);
+}
+
+- (void)testAcquireTokenSilent_whenATExpired_AndFailedToRefreshToken_shouldReturnError_AndRemoveExpiredAccesstoken_AndKeepRefreshTokenInCache
+{
+    MSIDRequestParameters *silentParameters = [self silentRequestParameters];
+    MSIDDefaultTokenCacheAccessor *tokenCache = self.tokenCache;
+
+    [self saveExpiredTokensInCache:tokenCache configuration:silentParameters.msidConfiguration];
```

Comment:
nit nit: quick test method to test if this actually works? I think multiple times, we rely on this being working yet I don't think we test this.

Reply:
I checked implementation and it's using our accessor's method saveTokensWithConfiguration, which is covered with unit tests.